docs/api/alerts/find.asciidoc

```diff
@@ -49,6 +49,8 @@ Retrieve a paginated set of alerts based on condition.
 NOTE: As alerts change in {kib}, the results on each page of the response also
 change. Use the find API for traditional paginated results, but avoid using it to export large amounts of data.
 
+NOTE: Alert `params` are stored as {ref}/flattened.html[flattened] and analyzed as `keyword`.
```

Comment:
@gchaps let me know what you think, I wanted to state that filtering on the `params` field has limitations.

Reply:
I've merged this PR, but I will create a follow-up if you would like changes to this line.